styleguide/_themes/global/scss/forms.scss

```diff
@@ -37,12 +37,12 @@ table {
   .help-block,
   .radio,
   .radio-inline {
-    color: #eb0022;
+    color: $brand-danger;
   }
 
   .form-control {
     background-color: rgba(235, 0, 34, .1);
```

Comment:
Yeah probably would be a good idea to at least get @bighappyface and @SyntaxStacks's feedback on this.